gluoncv/data/pascal_voc/detection.py

```diff
@@ -126,6 +131,11 @@ def _validate_label(self, xmin, ymin, xmax, ymax, width, height):
         assert xmin < xmax <= width, "xmax must in (xmin, {}], given {}".format(width, xmax)
         assert ymin < ymax <= height, "ymax must in (ymin, {}], given {}".format(height, ymax)
 
+    def _validate_class_names(self, class_list):
+        """Validate class names."""
+        assert all(" " not in c for c in class_list), "spaces"
```

Comment:
I think spaces are still good?

Reply:
The `strip()` results in any class names with spaces would be ignored too, so I added an assertion for that also

https://github.com/dmlc/gluon-cv/blob/428ee05d7ae4f2955ef00380a1b324b05e6bc80f/gluoncv/data/pascal_voc/detection.py#L106

Reply:
`strip()` removes whilespaces at the begening and end, won't affect those `abc def` class names

Reply:
you can probably do 

```
stripped = [c for c in class_list if c.strip() != c]
if stripped:
    warnings.warn('while space removed for {}'.format(stripped))
```

Reply:
This looks good! Changing